internal/serialization/portable_serializer_test.go

```diff
@@ -229,8 +285,34 @@ func TestPortableSerializer2(t *testing.T) {
 	data, _ := service.ToData(expectedRet)
 	ret, _ := service.ToObject(data)
 
+	expectedRet2 := &fake{byt, boo, ui16, i16, i32, i64, f32, f64, utf, nil,
+		byt_arr, boo_arr, ui16_arr, i16_arr, i32_arr, i64_arr, f32_arr, f64_arr, utf_arr, portable_arr}
+
+	data2, _ := service.ToData(expectedRet2)
+	ret2, _ := service.ToObject(data2)
+
+	config2 := NewSerializationConfig()
+	config2.AddPortableFactory(2, &PortableFactory1{})
+
+	service2 := NewSerializationService(config2)
+	ret3, _ := service2.ToObject(data)
+
+	if !reflect.DeepEqual(ret, expectedRet) || !reflect.DeepEqual(ret2, expectedRet2) || !reflect.DeepEqual(ret3, expectedRet) {
+		t.Errorf("ReadObject() failed!")
+	}
+
+}
```

Comment:
As far as I understand, there are two tests here.
* One serialization service can serialize and deserialize two different portables without messing things up
* Two different serialization services can read and write the same portable
Can you split the different tests and name them accordingly?